services/debian/debian.tester.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const { isVPlusDottedVersionNClauses } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Debian package (default distribution, valid)')
+  .get('/apt.json')
+  .expectBadge({ label: 'debian', message: isVPlusDottedVersionNClauses })
```

Comment:
Its reasonably common for Debian packages to have some kind of suffix on the version e.g: https://api.ftp-master.debian.org/madison?f=json&s=testing&package=python-dev Is `apt` always going to be a version without a suffix? If not, can we go with `isVPlusDottedVersionNClausesWithOptionalSuffix` to make the test less brittle?

Reply:
I don't think there are any such guarantees, so done. See also #3378